build-monitor-plugin/pom.xml

```diff
@@ -166,7 +166,7 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>groovy-postbuild</artifactId>
-            <version>2.3.1</version>
+            <version>2.4</version>
             <optional>true</optional>
         </dependency>
```

Comment:
+1 for the suggestion from @dcendents  .
This should be replaced with:
```
        <dependency>
            <groupId>org.jenkins-ci.plugins</groupId>
            <artifactId>badge</artifactId>
            <version>1.2</version>
            <optional>true</optional>
        </dependency>
```
(adding `optional=true` to the dcendents's comment)

And the plugin name to test should be also replaced to `badge`:
https://github.com/jan-molak/jenkins-build-monitor-plugin/blob/v1.11%2Bbuild.201701152243/build-monitor-plugin/src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobViews.java#L18